utils/mariadb/patches/210-libcxx.patch

```diff
@@ -0,0 +1,22 @@
+--- a/storage/innobase/handler/ha_innodb.cc
```

Comment:
Is this part of the upstream repository or not? If not, can you please send it to them?

Reply:
Sent the second one. But this one fails on libstdcpp. What a pain...